wrappers/ios/Sources/Wrapper/Writer/ZXIBarcodeWriter.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 KURZ Digital Solutions GmbH
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#import <Foundation/Foundation.h>
+#import "ZXIFormat.h"
+#import "ZXICharset.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface ZXIBarcodeWriter : NSObject
+
+-(nullable CGImageRef)write:(NSString *)contents
+                      width:(int)width
+                     height:(int)height
+                     format:(ZXIFormat)format
+                      error:(NSError **)error;
+
+-(nullable CGImageRef)write:(NSString *)contents
+                      width:(int)width
+                     height:(int)height
+                     format:(ZXIFormat)format
+                    charset:(ZXICharset)charset
```

Comment:
Not sure what the requirements exactly are. I will have a talk with my colleagues about this and come back to this probably next week.

Reply:
That is still an open question.

Reply:
Could you be more specific about what you mean by the interface could change? Should the possibility to pass a charset be completely removed? Otherwise the enum could be adapted at any time.

Reply:
Maybe removed or replaced by a completely different set of constants. To get an idea where this might be going, see #332. I spent so much time in the last 4 weeks thinking about and taking care to not break existing code or make the transition smooth. So adding stuff to the public API comes at a considerable cost. If you need that feature right now, I'd like to understand what for. And if you don't, then I'd like it to see removed from the PR.

Reply:
I removed the charset api.